packages/common/src/pivotTable/pivotQueryResults.ts

```diff
@@ -767,6 +761,68 @@ export const pivotQueryResults = ({
             pivotColumnInfo: [],
         },
     };
-
     return combinedRetrofit(pivotData, getField, getFieldLabel);
 };
+
+export const pivotResultsAsCsv = (
```

Comment:
I created this method here, so we can also use it in the frontend for exporting on embedding (similar api)

Reply:
This code is simila to what you reviewed yesterday fro embedding (I copied it from there to ensure we cover all the edge cases we found yesterday)